examples/raylib/raylib.c

```diff
@@ -0,0 +1,200 @@
+/*
+	TMX usage example with Raylib
+
+    To enable support of .jpg and other image file formats,
+    uncomment required in textures.c and rebuild Raylib.
+    See "Development Platforms" section in Raylib wiki.
+*/
+
+#include <stdio.h>
+#include <stdlib.h>
+#include <string.h>
+#include "tmx.h"
+#include "raylib.h"
+
+#define fatal_error(str)  { fputs(str, stderr); goto errquit; }
+#define DISPLAY_H 480
+#define DISPLAY_W 640
+
+
+struct Color color;
```

Comment:
Raylib draw functions accept Color as a parameter;
nothing like `SDL_SetRenderDrawColor`.
Not the best solution but simple (also, visually).